src/main/java/com/Leapwork/Leapwork_plugin/model/RunItem.java

```diff
@@ -28,15 +28,15 @@ public RunItem(String caseTitle, String caseStatus, double elapsed, String sched
         this.caseStatus = caseStatus;
         elapsedTime = elapsed;
         classname = schedule;
-        failure = null;
```

Comment:
Without it JUnit report in post build actions will not show keyframes. Why is it removed?